pkg/gornir/gornir.go

```diff
@@ -16,6 +17,41 @@ type Gornir struct {
 	Logger    Logger     // Logger for the object
 }
 
+// New is a Gornir constructor. It is currently no different that new,
+// however is a placeholder for any future defaults.
+func New() *Gornir {
+	return new(Gornir)
+}
+
+// InventoryPlugin is an Inventory Source
+type InventoryPlugin interface {
```

Comment:
Do we need this interface with this pattern? To me it feels that it's not needed as the creation is independent from Gornir, like the Logger.

Reply:
Yeah, probably not right now. I'm just trying to put a stake in the ground for all new Inventory sources, that they should provide a Create function that returns an `Inventory`.

Also for testing, I want to create a couple of hosts manually and pass that as a list to a Create function, so you end up with an `Inventory`.

Having said that, I agree with you and is not necessary at this point in time.